bliss/catalog.py

```diff
@@ -390,6 +413,25 @@ def union(self, other, disjoint=False):
                 d[k] = torch.where(ns11 > 0, d1, d2)
         return TileCatalog(d)
 
+    def stack(self, other):
```

Comment:
Is this different than the merge  method?

Reply:
The key of `stack` is to add the `n_sources_mask`. This variable will be very helpful in multidetection. Since the sources in tensor are sparse after using this function, I didn't call it union.

Reply:
I'm reluctant to add new methods to the main bliss package that are used solely in one case study. Could you keep this function outside of the TileCatalog class, in the case study where it's used, instead?

Reply:
get it

Reply:
remove the stack